src/odoc/indexing.mli

```diff
@@ -4,13 +4,14 @@ val handle_file :
   Fpath.t ->
   unit:(Odoc_model.Lang.Compilation_unit.t -> 'a) ->
   page:(Odoc_model.Lang.Page.t -> 'a) ->
+  occ:(Odoc_search.Entry.t Odoc_model.Paths.Identifier.Hashtbl.Any.t -> 'a) ->
```

Comment:
I don't remember, is `handle_file` exposed for sherlodoc?

Reply:
Yes it is used in sherlodoc. I would like that we do not remove it for now, sherlodoc still accepts .odocl files using this function.